openmrs/src/test/java/org/motechproject/openmrs/it/version1_12/MRSTaskIntegrationBundleIT.java

```diff
@@ -289,9 +284,5 @@ private boolean checkIfProgramEnrollmentWasCreatedProperly() {
     private void activateTrigger() {
         settingsDataService.create(new SettingsRecord());
     }
-
-    private void deleteTask(Long taskID) {
```

Comment:
There is a problem with deleting task after test execution. When action was executed successfully task activity is stored in database and after checking it test ends and task is deleted, but actually task still wants to store some data in task record, despite test has executed properly. The one way of solving this is to add timeout before deleting the task, but I've realised, that deleting task is only needed while debugging test several times locally because database is cleared before test so I just deleted it.